subsys/bluetooth/controller/ll_sw/nordic/lll/lll_prof.c

```diff
@@ -23,22 +23,22 @@
 
 static int send(struct node_rx_pdu *rx);
 static inline void sample(uint32_t *timestamp);
-static inline void delta(uint32_t timestamp, uint8_t *cputime);
+static inline void delta(uint32_t timestamp, uint16_t *cputime);
 
 static uint32_t timestamp_radio;
 static uint32_t timestamp_lll;
 static uint32_t timestamp_ull_high;
 static uint32_t timestamp_ull_low;
-static uint8_t cputime_radio;
-static uint8_t cputime_lll;
-static uint8_t cputime_ull_high;
-static uint8_t cputime_ull_low;
-static uint8_t latency_min = (uint8_t) -1;
-static uint8_t latency_max;
-static uint8_t latency_prev;
-static uint8_t cputime_min = (uint8_t) -1;
-static uint8_t cputime_max;
-static uint8_t cputime_prev;
+static uint16_t cputime_radio;
+static uint16_t cputime_lll;
+static uint16_t cputime_ull_high;
+static uint16_t cputime_ull_low;
+static uint16_t latency_min = UINT16_MAX;
+static uint16_t latency_max;
+static uint16_t latency_prev;
+static uint16_t cputime_min = UINT16_MAX;
+static uint16_t cputime_max;
+static uint16_t cputime_prev;
```

Comment:
Do all of these actually store valid values > 0xFF?

Reply:
CPU times are definitely higher these days and some values just over the 255 values. 
Under invalid behaviors use of uint16_t is necessary.